tests/integ/modin/series/test_quantile.py

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
I remember there was a test somewhere either in qcut/quantile where q was large and it was commented because of too many UNIONs. Do you think we can find & reactivate it? I would be curious to see if we can now with this change also support scenarios like `q=100` in qcut, or what a limit for `q` is we should set.

Reply:
I believe that's `test_quantile_large`, which I've un-XFAILed. I added a separate `test_quantile_large` for a frame with multiple columns in `frame/test_describe.py` which remains XFAILed since it doesn't benefit from this PR.

We could probably increase the limit for q as you suggested, but we'd have to have a separate limit for single-column frames vs. multi-column ones.

Reply:
perfect!